tariff/helper.go

```diff
@@ -28,19 +25,6 @@ func bo() backoff.BackOff {
 	)
 }
 
-// backoffPermanentError returns a permanent error in case of HTTP 400
-func backoffPermanentError(err error) error {
-	if se := new(request.StatusError); errors.As(err, &se) {
-		if code := se.StatusCode(); code >= 400 && code <= 599 {
```

Comment:
It seems the only error we don't want to treat as permanent is really 429?

Reply:
Why would we want to not retry 429s? Ideally we would increase the timeout between requests for those in particular, but since we are using exponential backoff anyways we will do the right thing.
In terms of status codes that should NOT be retried I am thinking of the ones where we don´t expect a retry to work:
```
400 Bad Request
401 Unauthorized
403 Forbidden
404 Not Found
501 Not Implemented
```
But in terms of listing these it´s probably easier to include the full range 400-599 and exclude the specific ones that services like grünstromindex sometimes return.